addons/main/modKonfigurationsTemplates/medicSettings/hardcore.h

```diff
@@ -1,15 +1,22 @@
 /* Hardcore */
-force force ace_medical_fatalDamageSource = 1; 					// Gibt es sofort t�dliche treffer? (Treffer in lebenswichtige Organe und den Kopf)
+
+// Gibt es sofort tödliche treffer? (Treffer in lebenswichtige Organe und den Kopf)
+#ifdef RR_MEDICSYSTEM_HARDCORE_DISABLE_FATAL_DAMAGE
+    force force ace_medical_fatalDamageSource = 0;
+#else
+    force force ace_medical_fatalDamageSource = 1;
+#endif
+
 force force ace_medical_limping = 2;							// Man kann einstellen, ob man durch Wunden humpelt und wenn ja, ob Bandagieren das Humpeln stoppt
-force force ace_medical_fractures = 1; 							// Gibt es Knochenbr�che? (aka braucht man Schienen und wie viel bringen sie?)
-force force ace_medical_treatment_medicEpinephrine = 1; 		// Wer darf Epi benutzen?
+force force ace_medical_fractures = 1; 							// Gibt es Knochenbr�che? (aka braucht man Schienen und wie viel bringen sie?)
+force force ace_medical_treatment_medicEpinephrine = 0; 		// Wer darf Epi benutzen?
 force force ace_medical_treatment_medicPAK = 1; 				// Wer darf PAKs benutzen?
-force force ace_medical_treatment_medicSurgicalKit = 1; 		// Wer darf das Surgical Kit (N�hzeug) benutzen?
```

Comment:
Why is the encoding touched? Either fix the encoding or don't touch it.

Reply:
Wouldn't how to fix that